actors/common/src/main/java/org/sunbird/learner/actors/syncjobmanager/EsSyncBackgroundActor.java

```diff
@@ -215,8 +215,7 @@ private String getType(String objectType) {
     userMap.put(JsonKey.BATCHES, Util.getUserCourseBatch(userId));
     try {
       userMap.put(
-          JsonKey.ROOT_ORG_NAME,
-          Util.getRootOrgIdOrNameFromChannel((String) userMap.get(JsonKey.CHANNEL), true));
+          JsonKey.ROOT_ORG_NAME, Util.getRootOrgName((String) userMap.get(JsonKey.ROOT_ORG_ID)));
```

Comment:
getRootOrgName instead of getting name only, u write a method to take orgId and return orgObject  and then u filter out response.